code/API_definitions/qod-api.yaml

```diff
@@ -1220,6 +1229,61 @@ components:
             message: "Service unavailable"
 
   examples:
+    SESSION_AVAILABLE_EXAMPLE:
+      summary: QoS session status is available
+      value:
+        duration: 86400
+        device:
+          ipv4Address:
+            publicAddress: 203.0.113.0
+            publicPort: 59765
+        applicationServer:
+          ipv4Address: 198.51.100.0/24
+        qosProfile: QCI_1_voice
```

Comment:
I personally would prefer to use here one of the predefined profiles, e.g. `QOS_L` ... not the specific example from QoS Profiles ("Support for predefined profiles QOS_S, QOS_M, QOS_L, and QOS_E")